PreFreeSurfer/scripts/T2wToT1wDistortionCorrectAndReg.sh

```diff
@@ -325,52 +369,74 @@ for TXw in $Modalities ; do
-    
+
     # Copy files to specified destinations
-    if [ $TXw = T1w ] ; then 
+
+    echo "      ... Copying files"
+    if [ $TXw = T1w ] ; then
        ${FSLDIR}/bin/imcp ${WD}/FieldMap2${TXwImageBasename}_Warp ${OutputT1wTransform}
        ${FSLDIR}/bin/imcp ${WD}/${TXwImageBasename} ${OutputT1wImage}
        ${FSLDIR}/bin/imcp ${WD}/${TXwImageBrainBasename} ${OutputT1wImageBrain}
     fi
-    
+
 done
 
 ### END LOOP over modalities ###
 
+if [ "${T2wImage}" = "NONE" ] ; then
+  echo ""
+  # ceho " ---> Skipping T2w to T1w registration"
+  echo " ---> Skipping T2w to T1w registration"
 
-### Now do T2w to T1w registration
-mkdir -p ${WD}/T2w2T1w
-    
-# Main registration: between corrected T2w and corrected T1w
-${FSLDIR}/bin/epi_reg --epi=${WD}/${T2wImageBrainBasename} --t1=${WD}/${T1wImageBasename} --t1brain=${WD}/${T1wImageBrainBasename} --out=${WD}/T2w2T1w/T2w_reg
-    
-# Make a warpfield directly from original (non-corrected) T2w to corrected T1w  (and apply it)
-${FSLDIR}/bin/convertwarp --relout --rel --ref=${T1wImage} --warp1=${WD}/FieldMap2${T2wImageBasename}_Warp.nii.gz --postmat=${WD}/T2w2T1w/T2w_reg.mat -o ${WD}/T2w2T1w/T2w_dc_reg
-    
-${FSLDIR}/bin/applywarp --rel --interp=spline --in=${T2wImage} --ref=${T1wImage} --warp=${WD}/T2w2T1w/T2w_dc_reg --out=${WD}/T2w2T1w/T2w_reg
-   
-# Add 1 to avoid exact zeros within the image (a problem for myelin mapping?)
-${FSLDIR}/bin/fslmaths ${WD}/T2w2T1w/T2w_reg.nii.gz -add 1 ${WD}/T2w2T1w/T2w_reg.nii.gz -odt float
-
-# QA image
-${FSLDIR}/bin/fslmaths ${WD}/T2w2T1w/T2w_reg -mul ${T1wImage} -sqrt ${WD}/T2w2T1w/sqrtT1wbyT2w -odt float
-    
-# Copy files to specified destinations
-${FSLDIR}/bin/imcp ${WD}/T2w2T1w/T2w_dc_reg ${OutputT2wTransform}
-${FSLDIR}/bin/imcp ${WD}/T2w2T1w/T2w_reg ${OutputT2wImage}
+else
+        
+  echo ""
+  # ceho " ---> Running T2w to T1w registration"
+  echo " ---> Running T2w to T1w registration"
+
+  ### Now do T2w to T1w registration
+  mkdir -p ${WD}/T2w2T1w
+
+  # Main registration: between corrected T2w and corrected T1w
+  echo "      ... Corrected T2w to T1w"
+  ${FSLDIR}/bin/epi_reg --epi=${WD}/${T2wImageBrainBasename} --t1=${WD}/${T1wImageBasename} --t1brain=${WD}/${T1wImageBrainBasename} --out=${WD}/T2w2T1w/T2w_reg
+
+  # Make a warpfield directly from original (non-corrected) T2w to corrected T1w  (and apply it)
+  echo "      ... Making a warpfield from original"
+  ${FSLDIR}/bin/convertwarp --relout --rel --ref=${T1wImage} --warp1=${WD}/FieldMap2${T2wImageBasename}_Warp.nii.gz --postmat=${WD}/T2w2T1w/T2w_reg.mat -o ${WD}/T2w2T1w/T2w_dc_reg
+  echo "      ... Applying warpfield"
+  ${FSLDIR}/bin/applywarp --rel --interp=spline --in=${T2wImage} --ref=${T1wImage} --warp=${WD}/T2w2T1w/T2w_dc_reg --out=${WD}/T2w2T1w/T2w_reg
+
+  # Add 1 to avoid exact zeros within the image (a problem for myelin mapping?)
+  echo "      ... Adding 1"
```

Comment:
I think some/many of the added echos are more verbose than necessary, but this one is particularly egregious.  They also seem to be throwing off github's diffs, not aligning the command from the original script with its counterpart in the PR.

Reply:
To be clear, this instance won't line up because its indentation is different and it is inside a conditional, but see https://github.com/Washington-University/HCPpipelines/pull/133/files#diff-1c050b68ec3e755e9c45c9056f86b6f0R380 for a line that would have lined up, except the extra newline and echo somehow tricked it.

Reply:
The echos were added as they proved valuable in tracking the progress. But we can remove any that seem to be over detailed. In terms of the diff, it might be thrown as this part is now nested within the if/else/fi construct.

Reply:
I see. Would that be an issue? It's just a more detailed tracking of progress.

Reply:
At the moment, it may make it take longer to review the PR, due to git thinking there are changes where there aren't any.  I don't have strong opinions on script verbosity, as long as it isn't to the point of silliness, incomprehensibility, or hundreds of lines per second.  Not sure what others' opinions are...

Reply:
If you want to ignore whitespace changes, you can click on the widget symbol and select the "Hide whitespace changes" option.  As regards the `echo` commands, we aren't entirely consistent, but the general goal/style is that output to stdout should be accomplished through the `log_Msg` command.

Reply:
The reason for verbose use od echo and doing it instead of `log_Msg` is to have the output more readable. Like illustrated in the attached image.
![ceho_and_verbosity](https://user-images.githubusercontent.com/1247439/65816032-31315880-e1f7-11e9-9c06-c778092e61cf.png)

Perhaps this is something that we can control through the "debug.shlib" as discussed previously?

Reply:
I am okay with echos for meaningful steps, but adding 1 is probably not in that category.